test/beforeHeadersSend.test.js

```diff
@@ -16,7 +17,7 @@ var URL = window.URL;
 const EXAMPLE_HREF = "https://www.example.com";
 const beforeSendHeaders = workflow.__get__('beforeSendHeaders');
 const setConfig = workflow.__get__('setConfig');
-const b64EncodedToken = "eyJ0eXBlIjoiUmVkZWVtIiwiY29udGVudHMiOltbMjQsNjIsNTYsMTAyLDc2LDEyNywyMDEsMTExLDE2MSwyMTgsMjQ5LDEwOSwzNCwxMjIsMTYwLDIxOSw5MywxODYsMjQ2LDEyLDE3OCwyNDksMjQxLDEwOCw2OSwxODEsNzcsMTQwLDE1OCwxMywyMTYsMTg0XSxbMjI3LDExLDk1LDIxNSwxNSwyMTUsMTM1LDI0LDEzNywxNzQsMjMzLDgsODYsMTQ4LDEzMCwxOTEsNDYsMTgzLDkyLDEwOCwxNjAsMjQ5LDE1OCwyMzEsMTU5LDIxOCwyNTQsODAsMTQ4LDQ0LDI5LDI1NF1dfQ==";
+const b64EncodedToken = "eyJ0eXBlIjoiUmVkZWVtIiwiY29udGVudHMiOltbMjQsNjIsNTYsMTAyLDc2LDEyNywyMDEsMTExLDE2MSwyMTgsMjQ5LDEwOSwzNCwxMjIsMTYwLDIxOSw5MywxODYsMjQ2LDEyLDE3OCwyNDksMjQxLDEwOCw2OSwxODEsNzcsMTQwLDE1OCwxMywyMTYsMTg0XSxbMjI3LDExLDk1LDIxNSwxNSwyMTUsMTM1LDI0LDEzNywxNzQsMjMzLDgsODYsMTQ4LDEzMCwxOTEsNDYsMTgzLDkyLDEwOCwxNjAsMjQ5LDE1OCwyMzEsMTU5LDIxOCwyNTQsODAsMTQ4LDQ0LDI5LDI1NF0sWzEyMywzNCw5OSwxMTcsMTE0LDExOCwxMDEsMzQsNTgsMzQsMTEyLDUwLDUzLDU0LDM0LDQ0LDM0LDEwNCw5NywxMTUsMTA0LDM0LDU4LDM0LDExNSwxMDQsOTcsNTAsNTMsNTQsMzQsNDQsMzQsMTA5LDEwMSwxMTYsMTA0LDExMSwxMDAsMzQsNTgsMzQsMTA1LDExMCw5OSwxMTQsMTAxLDEwOSwxMDEsMTEwLDExNiwzNCwxMjVdXX0=";
```

Comment:
Size of tokens has increased significantly, this could be an overhead in some scenarios. Any thoughts about token's size?

Reply:
@armfazh That is true, the size increases by over 200 bytes because I the base-64 encoding is called over the string representation of the byte array of the curve settings (which is only 53 bytes in fact).

We could significantly reduce this by encoding each of the byte arrays into hexidecimal before calling btoa() on it. I think this is worthwhile doing but it will also require a change to the data format that is sent by the client, so maybe it would be better suited to a separate change?

Reply:
I agree, this is a separate issue.